libcaf_core/caf/send.hpp

```diff
@@ -54,6 +54,16 @@ void unsafe_send_as(Source* src, const Dest& dest, Ts&&... xs) {
                                                std::forward<Ts>(xs)...);
 }
 
+template <message_priority P = message_priority::normal, class Dest,
+          class... Ts>
+void unsafe_anon_send(const Dest& dest, Ts&&... xs) {
+  static_assert(sizeof...(Ts) > 0, "no message to send");
```

Comment:
Just to check: this is unsafe because we don't check if the receiver accepts given message?

Reply:
Yes, you can pass in `strong_actor_ptr` and it will just send the message even without knowing whether it's a typed actor or not.